translations/generateLanguageFiles.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+MODULE_NAMES=${1:-"jaspTestModule"}
+PATH_TO_R_FILE=$2
+LANGUAGE_CODES=${3:-"nl de es pt ja tr"}
+
+create_file_if_it_doesnt_exist() {
+	if [ ! -f "$1" ]; then
+		touch "$1"
+	fi
+}
+
+do_lupdate()	{	lupdate -locations none -extensions cpp,qml -recursive $1 -ts $2;	}
+do_msgattrib()	{	msgattrib --no-obsolete --no-location ${1}/po/QML-${1}.pot -o $2;	}
+
+# in case people try this at home
+validate_module()
+{
+	if [ ! -f "$1/inst/qml" ] | [ ! -f "$1/R" ]; then
+		echo "either $1/inst/qml or $1/R doesn't exist, so $1 is probably not a JASP module."
+	fi
+}
+
+for moduleName in ${MODULE_NAMES[@]}
```

Comment:
The for-loop is here not anymore needed: there will be only one module

Reply:
The loop allows us to run this locally on multiple modules like so:
```
./jasp-actions/translations/generateLanguageFiles.sh "jaspDescriptives jaspTTests" jasp-actions/translations "nl de es pt ja tr" jasp-actions/translations
```
if that's not necessary I'll remove it.